va/va_cp.h

```diff
@@ -0,0 +1,160 @@
+/*
```

Comment:
shouldn't be, this header file, included in va.h ??

Reply:
This header is included in va.h @line4231